samples/README.DockerHub.md

```diff
@@ -27,22 +29,37 @@ docker run -it --rm -p 8000:80 --name aspnet_sample microsoft/dotnet-framework-s
 
 After the application starts, navigate to `http://localhost:8000` in your web browser. You need to navigate to the application via IP address instead of `localhost` for earlier Windows versions, which is demonstrated in [View the ASP.NET app in a running container on Windows](https://github.com/microsoft/dotnet-framework-docker/blob/master/samples/aspnetapp/README.md#view-the-aspnet-app-in-a-running-container-on-windows).
 
+### Container sample: Run a WCF service application
+
+Type the following command to run a sample WCF service application with Docker:
+
+```console
+docker run -it --rm --name wcfservice_sample microsoft/dotnet-framework-samples:wcfservie
+```
+After the container starts, you can find IP address of the container and then hookup the client application to the service. See [Try pre-built WCF Docker Images](https://github.com/microsoft/dotnet-framework-docker/blob/master/samples/wcfapp/README.md#try-pre-built-wcf-docker-images) for more details.
```

Comment:
I think it would be beneficial to have the complete instructions for using the wcfclient image to communicate with the service here.